### ISSUE TITLE
Continuous input disabled after "input_events" run

### ISSUE THREAD

**Issue by MadeinDave** [emoon/minifb#97]
Hello, after executing and closing the sample `input_events.c` on a Linux system, it seems to disable the continuous keyboard events, resulting in helding a keyboard key produce only a single input, not repeated.

**Comment by thoughton** [comment]
I've encountered this too on a very simple project of my own. Interestingly I only seem to encounter the issue when compiling as `C++`; when compiling a _virtually_ identical `C` project I do not get the problem.

**Comment by Darky-Lucera** [comment]
It sounds weird. X11 or Wayland?

**Comment by thoughton** [comment]
Yeah X11 here too. Ubuntu 22.10 in my case.

Also I've disproven my earlier claim about it being different between C/C++ so I'll edit my previous reply.

**Comment by thoughton** [comment]
This issue has been confusing me so much, because as I mentioned I have 2 very similar projects, 1 working and 1 broken.

And then last night I finally realised the obvious difference: they were started at different times, so they had different revisions of `minifb` synced.

I confirmed this theory by copying the `minifb` from the older working project to the newer broken project, and the newer project started working too.

I haven't had time to track down exact revisions, but I have noted the following:
```
db8ffbb02809ff01205beac370d78d969bd7ab56 = Broken

cc79a85e3d101e6b4a43a61193739fa0299f3a60 = Working
```
